R/as_rtf.R

```diff
@@ -0,0 +1,101 @@
+#' Print a summary table using gt
+#'
+#' Create RTF file created with \code{\link{as_table}} to summarize
+#' an object; currently only implemented for \code{\link{gsBinomialExact}}.
+#'
+#' @param x   Object to be display on rtf.
+#' @param ... Other parameters that may be specific the object.
+#'
+#' @return NULL.
```

Comment:
You should return the input object invisibly. See tidyverse design guide: https://design.tidyverse.org/out-invisible.html

Reply:
Note that the returned object should be the **original input**, not the modified object.